addons/website_blog/static/src/js/website_blog.js

```diff
@@ -92,11 +97,11 @@ publicWidget.registry.websiteBlog = publicWidget.Widget.extend({
 
     /**
      * @private
-     * @param {JQuery} $el - the element we are scrolling to
+     * @param {HTMLElement} el - the element we are scrolling to
      * @param {Integer} duration - scroll animation duration
      * @param {Function} callback - to be executed after the scroll is performed
      */
-    _forumScrollAction: function ($el, callback) {
-        scrollTo($el[0], { behavior: "smooth" }).then(() => callback());
+    _forumScrollAction: function (el, duration, callback) {
+        scrollTo(el, {duration: duration}).then(() => callback());
```

Comment:
I think you missed an update during a rebase; the diff does not correspond ;)

Reply:
That still not up to date :face_with_peeking_eye:

Reply:
oops i didnt changed on rebase, i guess i need to do it manually!